src/components/Composer.vue

```diff
@@ -123,17 +123,26 @@
 			></TextEditor>
 		</div>
 		<div class="composer-actions">
-			<ComposerAttachments v-model="attachments" @upload="onAttachmentsUploading" />
+			<ComposerAttachments ref="composerAttachments" v-model="attachments" @upload="onAttachmentsUploading" />
 			<div class="composer-actions-right">
 				<p class="composer-actions-draft">
 					<span v-if="savingDraft === true" id="draft-status">{{ t('mail', 'Saving draft …') }}</span>
 					<span v-else-if="savingDraft === false" id="draft-status">{{ t('mail', 'Draft saved') }}</span>
 				</p>
 				<Actions>
-					<ActionText icon="icon-info">{{ t('mail', 'Message options') }}</ActionText>
-					<ActionCheckbox :checked.sync="editorPlainText" :text="t('mail', 'Plain text')">{{
-						t('mail', 'Plain text')
-					}}</ActionCheckbox>
+					<ActionButton icon="icon-upload" @click="onAddLocalAttachment">{{
+						t('mail', 'Upload attachment')
+					}}</ActionButton>
+					<ActionButton icon="icon-folder" @click="onAddCloudAttachment">{{
+						t('mail', 'Add attachment from Files')
+					}}</ActionButton>
+					<ActionCheckbox
+						:checked="!editorPlainText"
+						:text="t('mail', 'Enable text formatting')"
```

Comment:
Please revert to the original message.

Moving the checkbox is okay, but sneaking in a new label with an unrelated change is a bit meh. Did @jancborchardt talk you into this? :D

Reply:
hmm, @jancborchardt and me thought it would make sense to change this, because a checkbox saying "plain text" which reveals formatting possibilities when unchecked is only understandable for techie users. For accessibility reasons it would make sense to describe features in clear and precise language...
but of course if you want me to revert this I can do so or I could also create a separate PR to discuss this there if you prefer...?

Reply:
It was @violoncelloch's suggestion and I agree with it, also saying it could be added here since we talked about it in the same context. Yes it should be in a separate PR, my mistake. :)

Reply:
Yes, @violoncelloCH, please do this in a separate PR. I'm fine with the change, just not when the PR is about *put composer attachment buttons into three dot menu* :wink:

Reply:
let me do that :)